scripts/ci/fix-locales-and-push-changes

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+fix() {
+  ./scripts/ci/fix-weblate public/locales/*/*.json
+  npx prettier --write public/locales/*/*.json
```

Comment:
Doesn't this happen in git commit?

Reply:
Yes, good point, although it doesn't actually work when I tried it here... I just managed to commit:

```json
{
  "Polish":           "Polština",
     "foo": "bar"
}
```
locally... and no fixing occured :/

... digdigdig, `.eslintignore` says to ignore `public`...  we re-use the ignore file from eslint for prettier, _but_ they have different syntaxes/semantics (a bit)...

- https://eslint.org/docs/user-guide/configuring#eslintignore
- https://git-scm.com/docs/gitignore#_pattern_format

working on it...

Reply:
Fixed in the last few commits... oops also seeing I set my local git credentials to the Trustroots CI user :)